tests/check/check_h26xsigned_sign.c

```diff
@@ -600,6 +600,7 @@ START_TEST(two_completed_seis_pending)
   size_t sei_size_2 = 0;
   size_t sei_size_3 = 0;
   signed_video_t *sv = signed_video_create(codec);
+  // Enable testing mode to add multiple SEIs.
```

Comment:
Implicitly this is correct, but there is no such mode in the code. Rephrase this. For example that this will bypass a check so multiple NAL Units can be added before getting SEIs